features/foot/restrictions.feature

```diff
@@ -29,7 +29,7 @@ Feature: Foot - Turn restrictions
         When I route I should get
             | from | to | route    |
             | s    | w  | sj,wj,wj |
-            | s    | n  | sj,nj,nj |
+            | s    | n  | sj,nj    |
```

Comment:
This test uses foot.lua profile, which in its turn uses `suffix_list` defined as `Set`. That's why fix of the bug with Set values made this test red.

https://github.com/Project-OSRM/osrm-backend/blob/06b1b980bbdfc838c4f8e8cb370cba6dfb0329b0/profiles/foot.lua#L67

Reply:
Hmm, it's not immediately obvious why this would change due to the bug fix, unless it's something very subtle in the code.